e2e/cypress/integration/messaging/emoji_size_spec.js

```diff
@@ -84,4 +84,26 @@ describe('Messaging', () => {
         // # Making sure Emojis from last post message are 21px size
         checkEmojiSize('@newLineMessage', emojis, false);
     });
+
+    it('MM-17457 Unicode and emoticon', () => {
+        // # Create list of emojis we want to post
+        const emojis = [':smiley:', ':thumbsup:', '🤟'];
+
+        // # Post Emojis list
+        cy.postMessage(emojis.join(''));
+
+        // #Get last post message
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).as('lastMessage');
+
+            //# Expect unicode value from last message to have jumbo size
+            cy.get('@lastMessage').find('.emoticon--unicode').should('have.css', 'height', '32px').and('have.css', 'width', '32px');
```

Comment:
``.and('have.text', '🤟')`` --> Please include additional check.

Reply:
Done